### ISSUE TITLE
Deprecate repo

### ISSUE THREAD

**Issue by hunner** [urbit/hoon.vim#10]
The description for this repository should probably be updated to point users to https://github.com/urbit/urbit/tree/master/extras/hoon.vim (unless of course this repo is still relevant).

**Comment by galenwp** [comment]
I think we were hoping to move towards a less monolithic repo, so I'd vote to keep this one up to date.  But this is really @philipcmonk territory.  Thoughts, Philip?

**Comment by hunner** [comment]
You know, on second thought after messing with this and hoon-mode.el, editor plugins in separate repos a) makes it easy for plugin managers to just clone them down without worrying about sub-sub-directories, and b) doesn't need all the files in the repo (eg, urbit/urbit is 22MB to get urbit/extras/hoon.vim).

So I guess this should actually be "deprecate the extras/ vim plugin in favor of this repo")

**Comment by philipcmonk** [comment]
Indeed, I would argue that the extras directory should be removed altogether (after ensuring that everything in it has another home).  This is not a part of Urbit core.

Work items would be:
- merge this and extras/hoon.vim (looks like extras/hoon.vim is slightly more up-to-date?)
- ensure hoon-mode.el is somewhere (create repo if necessary)
- remove from master
- Bonus:  update hoon.vim for the Hoon 151 language updates

**Comment by ohAitch** [comment]
+1

See also https://github.com/ohAitch/LT-Hoon for codemirrror bindings,
though those are actively used by /tree
